.github/workflows/pitest-analyze-pr.yml

```diff
@@ -7,6 +7,8 @@ on:
   pull_request:
 permissions:
   contents: read
+env:
+  MAVEN_ARGS: --batch-mode -Dstyle.color=always -T1C
```

Comment:
Previously we didn't parallelize the Pitest analysis; let's see whether we get away with this, resource utilization-wise.